frontend/packages/noobaa-storage-plugin/src/components/data-resiliency-card/data-resiliency-card.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
+import { Progress } from '@patternfly/react-core';
+
+import './data-resiliency-card.scss';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { DashboardCardHelp } from '@console/internal/components/dashboard/dashboard-card';
+
+import { DATA_RESILIENCE_QUERIES } from '../../queries';
+import { getPropsData } from '../../utils';
+
+const DataResiliencyStatusBody: React.FC<DataResiliencyStatusBody> = ({ isResilient }) =>
+  isResilient ? (
+    <>
+      <div className="nb-data-resiliency__status-title-ok">Your Data is Resilient</div>
+      <div className="nb-data-resiliency__icon-ok">
+        <Icon type="fa" name="check-circle" size="5x" />
+      </div>
+    </>
+  ) : (
+    <>
+      <div className="nb-data-resiliency__icon-error">
+        <Icon type="fa" name="exclamation-triangle" size="5x" />
+      </div>
+      <div className="nb-data-resiliency__status-title-error">No data available</div>
+    </>
+  );
+
+const DataResiliencyBuildBody: React.FC<DataResiliencyBuildBody> = ({
+  progressPercentage,
+  eta,
+}) => (
+  <>
+    <div className="nb-data-resiliency__title">Rebuilding data</div>
+    <Progress
+      className="nb-data-resiliency__utilization-bar"
+      value={progressPercentage}
+      title="Rebuilding in Progress"
+      label={`${progressPercentage}%`}
+    />
+    <span className="text-secondary nb-data-resiliency__eta">
+      Estimating {eta} mins to completion
```

Comment:
do we really want `mins` here ?

Reply:
I told you , dont have clarity over that, hence wip.
will be changing later